tools/src/main/python/dlpx/virtualization/_internal/codegen.py

```diff
@@ -108,12 +108,32 @@ def generate_python(name, source_dir, plugin_config_dir, schema_content):
     _copy_generated_to_dir(output_dir, source_dir)
 
 
+def _make_url_refs_opaque(json):
+    if isinstance(json, dict):
+        for key in json:
+            if key == '$ref' and isinstance(json[key], basestring)\
```

Comment:
I think I understand what this function is doing, but it took me a while. It maybe could use a short comment describing what it's doing.